lib/utils/validationHelper.js

```diff
@@ -1,6 +1,29 @@
+const crypto = require('crypto');
 const errors = require('./errors');
 const Datatypes = require('./datatypes');
 
+const validateApiKey = (appUuid, apiKey) => {
```

Comment:
```suggestion
const validateApiKey = (appId, apiKey) => {
```
Just for consistency

Reply:
I think for non-public facing variables, it's ok to have appUuid (just to make it less confusing as the API still expect an UUID -> we should change that)